plugins/unifying/lu-device.c

```diff
@@ -892,6 +919,10 @@ lu_device_open (LuDevice *device, GError **error)
 		return FALSE;
 	}
 
+	/* the guid generation is delayed until the bootloader string is read during probe */
+	if (devid != NULL)
+		lu_device_add_guid (device, devid);
```

Comment:
Rather than re-using devid, I think you want something like devid_bootver or something

Reply:
I don't think so, the bootloader version is actually appended inside `lu_device_add_guid` and devid only is the USB\\VID+PID at this point, right?